src/main/java/org/broadinstitute/hellbender/engine/dataflow/transforms/composite/AddContextDataToRead.java

```diff
@@ -47,7 +48,7 @@
             assertSameReads(pReads, kvReadRefBases, kvReadVariants);
         }
         // We could add a check that all of the reads in kvReadRefBases, pVariants, and pReads are the same.
-        PCollection<KV<UUID, Iterable<Variant>>> UUIDVariants = kvReadVariants.apply(ParDo.of(new DoFn<KV<GATKRead, Iterable<Variant>>, KV<UUID, Iterable<Variant>>>() {
+        PCollection<KV<UUID, Iterable<Variant>>> UUIDVariants = kvReadVariants.apply(ParDo.named("UUIDVariants").of(new DoFnWLog<KV<GATKRead, Iterable<Variant>>, KV<UUID, Iterable<Variant>>>("UUIDVariants") {
```

Comment:
Should these names for the `ParDos` (here and below) match the names in the `setName()` calls on the resulting `PCollections`